common/rpc/RequestBuilder.java

```diff
@@ -665,23 +665,23 @@ static ByteString byteString(String iid) {
                 ));
             }
 
-            public static ConceptProto.Attribute.Value attributeValueBooleanReq(boolean value) {
```

Comment:
These methods weren't actually constructing `Req` messages so we took Req out of the name.